pkg/components/wfbackend/metadata.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package wfbackend
+
+import (
+	"time"
+
+	"github.com/microsoft/durabletask-go/backend/sqlite"
+
+	"github.com/dapr/components-contrib/metadata"
+	metadataUtil "github.com/dapr/kit/metadata"
+)
+
+const (
+	defaultFilePath                 = ""
```

Comment:
These constants are specific for SQLite, so should have a prefix (unless you move the Metadata struct to a separate file per my other comment)